messages/edge_function/messages.go

```diff
@@ -45,17 +45,17 @@ var (
 	UpdateAskEdgeFunctionID = "What is the ID of the edge function you wish to update?"
 
 	// flags
-	FlagID     = "Unique identifier of the Edge Function"
-	FlagName   = "The Edge Function's name"
-	FlagCode   = "Path to the Edge Function's code"
-	FlagActive = "Whether the Edge Function is active or not"
-	FlagArgs   = "Path to the Edge Function's arguments JSON file"
-	FlagIn     = "Given file path to create an Edge Function; you can use - for reading from stdin"
-	FlagHelp   = "Displays more information about the edge function subcommand"
+	FlagID         = "Unique identifier of the Edge Function"
+	FlagName       = "The Edge Function's name"
+	FlagCode       = "Path to the Edge Function's code"
+	FlagActive     = "Whether the Edge Function is active or not"
+	FlagArgs       = "Path to the Edge Function's arguments JSON file"
+	FlagIn         = "Given file path to create an Edge Function; you can use - for reading from stdin"
+	CreateFlagHelp = "Displays more information about the create edge-function command"
 
 	// ask
 	AskName           = "What is the Name of the edge function?"
 	AskCode           = "What is the Code Path  of the edge function?"
-	AskActive         = "Does activating the edge function?"
+	AskActive         = "Select whether the edge function is active or not"
```

Comment:
I know i'm not a content writer, but shouldn't/couldn't we use the same pattern for this `ask` parameter like the others? 
Like:
```suggestion
	AskActive         = "What is the activation status of the edge function?"
```

Reply:
The parameter suggest that we are `Asking` something, not selecting.

Reply:
It's a list of options. I changed it to follow the same pattern of other list of items shown. 
Also, the lib puts a colon at the end of the sentence. It looks weird:
`What is the activation status of the edge function?:`

Reply:
What could be done is change this to a yes/no question as well, and manually transform this into true/false

Reply:
OR we could change the name of the var

Reply:
The list of patterns for `ask` parameters is not the one you changed, is the one i suggested using `What`.
For reference, see the other ask parameters already on code :
```
// ask
	AskName           = "What is the Name of the edge function?"
	AskCode           = "What is the Code Path  of the edge function?"
	AskEdgeFunctionID = "What is the ID of the edge function?"
```
This is why i made my suggestion. 
You are asking for a status, for an id....this is a doubt. 
For me it is strange `selecting` something that you only `ask` for a status. 
And for me, it makes sense use `Ask` for the var name.

Reply:
But bro, don't sweat it. 
This is a completely aesthetic suggestion. 😄 
Based on my preferences and understandings.
I trust your opinion for this great job you are doing.

Reply:
I also have my doubts about using a list of two items. We probably will change this going forward. 
@helio-neto every suggestion is important! 😄